src/client/dfuse/ops/create.c

```diff
@@ -91,8 +91,15 @@ dfuse_cb_create(fuse_req_t req, struct dfuse_inode_entry *parent,
 	oh->doh_dfs = parent->ie_dfs->dfs_ns;
 	oh->doh_ie = ie;
 
-	if (fi->direct_io)
-		fi_out.direct_io = 1;
+	if (!fs_handle->dpi_info->di_no_direct_io) {
```

Comment:
Just a minor nit.   This would read nicer if it were positive rather than a double negative

e.g. if (fs_handle->dpi_info->di_direct_io)

so it would require changing the option parsing above

Reply:
True, and I'm normally a strong advocate of not using double negatives, but in this case I thought the flag is really a workaround for a known issue rather than a feature flag.